wgsl/index.bs

```diff
@@ -5297,12 +5297,18 @@ See [[#discard-statement]].
     function parameter.
     * In particular, an argument that is a pointer must agree with the formal parameter
         on storage class, pointee type, and access mode.
-* A function parameter of pointer type must be in one of
+* For [=user-defined functions=], a parameter of pointer type must be in one of
     the following storage classes:
     * [=storage classes/function=]
     * [=storage classes/private=]
     * [=storage classes/workgroup=]
-* Each argument of a function call of pointer type must be one of:
+* For [=built-in functions=], a parameter of pointer type must be in one of
+    the following storage classes:
+    * [=storage classes/function=]
+    * [=storage classes/private=]
+    * [=storage classes/workgroup=]
+    * [=storage classes/storage=]
+* Each argument of pointer type to a [=user-defined function=] must be one of:
```

Comment:
Should this condition still apply to frexp and modf?
Looking for advice from @ben-clayton

Reply:
Resolution for frxp and modf is to use a struct form with unnameable struct type, and to remove the pointer forms. See #1480